### PR TITLE
POST explicit list of files to review endpoint

### PR DESCRIPTION
**Pull request by bloodearnest**
For now, it just posts all of them. Future changes will allow selection.

The backend validates and then generates a zip with just files the
requested.

The JS works in the simplest way possible, inserting the files when the
form is submitted. This plays very nicely with CSRF and
Content-Disposition downloads.

### CODE REVIEW

**Review by tomodwyer — APPROVED** [review]
JS looks fine

File: assets/src/scripts/_form-setup.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
   const form = document.querySelector("#approveForm");
   const button = form.querySelector(`button[type="submit"]`);
   form.action = reviewUrl;
 
+  form.addEventListener("formdata", (ev) => {
+    // TODO: get from signal
+    for (const [output] of outputs) {
+      ev.formData.append("outputs", output);
+    }
+  });
```
Comment (tomodwyer) [inline assets/src/scripts/_form-setup.js:13]:
```suggestion
    [...outputs].forEach(([output]) => ev.formData.append("outputs", output));
```

I'm not a huge fan of `for ... of` loops, so you could change to a `forEach` iterator.

Comment (bloodearnest) [reply assets/src/scripts/_form-setup.js:13]: Why the preference, OOI?

Comment (tomodwyer) [reply assets/src/scripts/_form-setup.js:13]: These two links probably explain it better than I can:

- https://github.com/airbnb/javascript/issues/1271#issuecomment-283735174
- https://github.com/airbnb/javascript/issues/1271#issuecomment-283736133

Although the flame war in that thread suggests I may be wrong.

**Pull request merged**